docs/photoniq/prerendering/get-started-prerendering.md

```diff
@@ -11,27 +11,27 @@ However, some people like to be more hands-on with their management. After you w
 
 Before leveraging the caching capabilities of Dynamic Prerendering service, you must first define your origin, which is the site that you want it to prerender. This step is crucial as it allows the system to understand the source of the pages and how they should be cached for optimal performance.
 
-- Make a request to the `/api/prerender/v1/origin` endpoint using the POST method.
+- Make a request to the [Creates an origin](/docs/apiPrerendering#/paths/api-prerender-v1-origins/post) endpoint using the POST method.
 - Include the necessary headers and provide a body containing your origin details.
 
 ## 2. Fetch Prerendered Page Content
 
-To retrieve content, use the`/api/prerender/v1/render` endpoint.
+To retrieve content, use the [Fetches the prerendered page content](/docs/apiPrerendering#/paths/api-prerender-v1-render-mobile---url/get) endpoint.
```

Comment:
Does this URL work with three dashes in a row?

Reply:
Yes, this is now pointing directly to the API reference page. I just copied the link from there, starting from `/docs`.